### PR TITLE
Fixes while trying to use extender

### PR DESCRIPTION
**Pull request by smagnuso**
Found some documentation issues
Also fixed the build-docker.sh to make it easier to pass in what things to install rather than commenting out bits that I do not have.

### CODE REVIEW

File: server/build-docker.sh [hunk 1]
```diff
@@ -15,28 +15,56 @@ echo "Wine base image"
...
+        ps4-*)
+            DM_PACKAGES_URL=$DM_PACKAGES_URL docker buildx build --secret id=DM_PACKAGES_URL --platform linux/amd64 -t $DOCKER_PS4_PRIVATE_REGISTRY/extender-${install}-env:latest -f $SCRIPT_DIR/docker/Dockerfile.$(echo $install | sed 's,-,.,')-env $SCRIPT_DIR/docker
+            ;;
+        ps5-*)
+            DM_PACKAGES_URL=$DM_PACKAGES_URL docker buildx build --secret id=DM_PACKAGES_URL --platform linux/amd64 -t $DOCKER_PS5_PRIVATE_REGISTRY/extender-${install}-env:latest -f $SCRIPT_DIR/docker/Dockerfile.$(echo $install | sed 's,-,.,')-env $SCRIPT_DIR/docker
+            ;;
+        nssdk-*)
+            DM_PACKAGES_URL=$DM_PACKAGES_URL docker buildx build --secret id=DM_PACKAGES_URL --platform linux/amd64 -t $DOCKER_NINTENDO_PRIVATE_REGISTRY/extender-${install}-env:latest -f $SCRIPT_DIR/docker/Dockerfile.$(echo $install | sed 's,-,.,')-env $SCRIPT_DIR/docker
+            ;;
+        android-ndk*|winsdk-*|emsdk-*|android)
```
[the remark this answers is not in the document]
Comment (smagnuso) [reply server/build-docker.sh:61]: Well spotted, thanks!

File: server/build-docker.sh [hunk 2]
```diff
@@ -15,28 +15,56 @@ echo "Wine base image"
 DM_PACKAGES_URL=$DM_PACKAGES_URL docker buildx build --secret id=DM_PACKAGES_URL --platform linux/amd64 -t $DOCKER_REGISTRY/extender-wine-env:1.2.1 -t $DOCKER_REGISTRY/extender-wine-env:latest -f $SCRIPT_DIR/docker/Dockerfile.wine-env $SCRIPT_DIR/docker
 echo "Linux image"
 DM_PACKAGES_URL=$DM_PACKAGES_URL docker buildx build --secret id=DM_PACKAGES_URL --platform linux/amd64 -t $DOCKER_REGISTRY/extender-linux-env:latest -f $SCRIPT_DIR/docker/Dockerfile.linux-env $SCRIPT_DIR/docker
```
[the remark this answers is not in the document]
Comment (smagnuso) [reply server/build-docker.sh:17]: Not sure I see the upside tbh, nor do I know which platforms it is relevant for. Maybe you can just make further modification after for that?

File: server/build-docker.sh [hunk 3]
```diff
@@ -15,28 +15,56 @@ echo "Wine base image"
 DM_PACKAGES_URL=$DM_PACKAGES_URL docker buildx build --secret id=DM_PACKAGES_URL --platform linux/amd64 -t $DOCKER_REGISTRY/extender-wine-env:1.2.1 -t $DOCKER_REGISTRY/extender-wine-env:latest -f $SCRIPT_DIR/docker/Dockerfile.wine-env $SCRIPT_DIR/docker
```
[the remark this answers is not in the document]
Comment (smagnuso) [reply server/build-docker.sh:15]: Sure, I don't mind. Did that while I fixed android above.

**Comment by smagnuso** [comment]
Not sure I feel necessary to add documentation, it is already confusing which paths to take in there to be honest. The currently documented path still works (ie no arguments) and this is just for those to avoid having to comment it out and dirty their tree.

File: server/build-docker.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (JCash) [reply server/build-docker.sh:15]: I agree, we can add it later.

**Review by JCash — APPROVED** [review]

**Pull request merged**